src/libs/actions/Policy.js

```diff
@@ -455,6 +455,75 @@ function update(policyID, values, shouldGrowl = false) {
         });
 }
 
+/**
+ * Optimistically update the workspace general settings. Set the general settings as pending until the response succeeds.
+ * If the response fails set a general error message. Clear the error message when updating.
+ *
+ * @param {String} policyID The id of the workspace / policy
+ * @param {String} name The new workspace name
+ * @param {String} currency The new workspace currency
+ */
+function updateWorkspaceGeneralSettings(policyID, name, currency) {
+    const optimisticData = [
+        {
+            onyxMethod: CONST.ONYX.METHOD.MERGE,
+            key: `${ONYXKEYS.COLLECTION.POLICY}${policyID}`,
+            value: {
+                pendingFields: {
+                    generalSettings: CONST.RED_BRICK_ROAD_PENDING_ACTION.UPDATE,
+                },
+
+                // Clear errorFields in case the user didn't dismiss the error
```

Comment:
`the error` : which error? 😕

Reply:
The general settings error. I'll clarify that.